WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsItem.swift

```diff
@@ -0,0 +1,201 @@
+                        .subheadlineStyle()
+                    Spacer()
+                    Image(systemName: "info.circle")
+                        .foregroundColor(Color(.wooCommercePurple(.shade60)))
+                }
+                .padding(.top, Layout.descriptionTopPadding)
+                TextField("", text: $viewModel.description)
+                    .padding(Layout.extraPadding)
+                    .roundedBorder(cornerRadius: Layout.borderCornerRadius, lineColor: Color(.separator), lineWidth: Layout.borderLineWidth)
+                Text(Localization.HSTarriffNumber)
+                    .foregroundColor(.primary)
+                    .subheadlineStyle()
+                TextField(Localization.HSTarriffNumberPlaceholder, text: $viewModel.hsTariffNumber)
+                    .padding(Layout.extraPadding)
+                    .roundedBorder(cornerRadius: Layout.borderCornerRadius, lineColor: Color(.separator), lineWidth: Layout.borderLineWidth)
+                Button {
+                    isShowingHSTarrifInfoWebView = true
+                } label: {
+                    HStack(alignment: .top, spacing: Layout.hsTarriffNumberMoreInfoVerticalSpacing) {
+                        Image(systemName: "info.circle")
+                        Text(Localization.HSTarriffNumberMoreInfo)
+                    }
+                    .foregroundColor(Color(.wooCommercePurple(.shade60)))
+                    .footnoteStyle()
+                }
+
+                HStack {
+                    VStack(alignment: .leading) {
+                        Text(Localization.valuePerUnitTitle)
+                            .foregroundColor(.primary)
+                            .subheadlineStyle()
+                        TextField("$ 0", text: $viewModel.valuePerUnit)
+                            .padding(Layout.extraPadding)
+                            .roundedBorder(cornerRadius: Layout.borderCornerRadius,
+                                           lineColor: viewModel.valuePerUnit.isEmpty ? .withColorStudio(name: .red, shade: .shade60) : Color(.separator),
+                                           lineWidth: Layout.borderLineWidth)
+                        Text(Localization.valueRequiredWarningText)
+                            .foregroundColor(.withColorStudio(name: .red, shade: .shade60))
+                            .footnoteStyle()
+                            .renderedIf(viewModel.valuePerUnit.isEmpty)
+                    }
+
+                    VStack(alignment: .leading) {
+                        Text(Localization.weightPerUnitTitle)
+                            .foregroundColor(.primary)
+                            .subheadlineStyle()
+                        HStack {
+                            TextField("0", text: $viewModel.weightPerUnit)
+                                .padding(Layout.extraPadding)
+                            // TODO: Add right unit
+                            Text("kg")
```

Comment:
FYI the weight unit is set as an environment value, so you could access it with a property like `@Environment(\.shippingWeightUnit) var weightUnit: String` in this view.

Reply:
Awesome, thanks for the hint, added here 2456a91c3fdf0470484438485bc59066363ccc83